docs/fileformat-prose.rst

```diff
@@ -0,0 +1,170 @@
+On-disk format
+--------------
+
+.. note::
+   These docs are written for anndata 0.7.
+   Files written before this version may differ in some conventions, but will still be read by newer versions of the library.
+
+AnnData objects are saved on disk to hierarchichal array stores like `HDF5 <https://en.wikipedia.org/wiki/Hierarchical_Data_Format>`_ (via `h5py <http://docs.h5py.org/en/stable/>`_) and `Zarr <https://zarr.readthedocs.io/en/stable/>`_.
```

Comment:
Why is that? I think it can be useful for when reading the un-rendered version, and for reducing references that are kept around but not used.

Reply:
I certainly have a much easier time reading things that aren’t a jumble of interspersed links and text. The above paragraph is just a word salad and I need significantly longer parsing out the actual text.

Reply:
Hmm, I guess I've always got the link colored differently in my editor. I also think it introduces the new problems of needing to come up with a link name myself, keeping those synced, and keeping them unique (though there might be tooling for this?).

Reply:
None of this has ever been a problem for me. I find that using the link text as ID works really well in practice and results in less useless text like “click here” (with “here” being a link). It just reads better.

Reply:
Oh, I've run into this on a number of open source projects. Mainly it's "this link makes no sense in this context", and it turns out there was either a namespace collision.

Is there any sort of scoping for these identifiers names? The [Rst docs](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#reference-names) make me it could be global.

Reply:
[`:ref:`](https://www.sphinx-doc.org/en/1.5/markup/inline.html#role-ref) allows to link cross-document, plain links don’t:

> Using `ref` is advised over standard reStructuredText links to sections (like `` `Section title`_``) because it works across files, when section headings are changed, and for all builders that support cross-references.